### PR TITLE
add docs for close

### PR DESCRIPTION
**Pull request by bAndie91**
sorry i left out documentation

### CODE REVIEW

**Comment by vi** [comment]
Shall it mention that `close` is called only in one of two modes (i.e. when `open` script returned some file)?

**Comment by bAndie91** [comment]
@vi, you are right. however, is not it be better to call close on both modes? what do you think?

File: execfuse.c [hunk 1]
```diff
@@ -506,6 +506,10 @@ static int execfuse_release(const char *path, struct fuse_file_info *fi)
 
     int ret = 0;
     
+    /* notify the script when closing a file */
+    call_script_simple("close", path);
+    /* ignore return code since fuse also ignores it, see https://libfuse.github.io/doxygen/structfuse__operations.html#a4a6f1b50c583774125b5003811ecebce */
+
```
[the remark this answers is not in the document]
Comment (bAndie91) [reply execfuse.c]: ye you are right. i have not thought about it.

it's a bit awkward that fuse does not recognize the return code of `release` call. maybe this is why it's called `release` and not `close`.

what's your opinion about the naming? shall the user script also called `release`, or in past `released` or `closed` quasi indicating that the user script gets called ater the operation (unlike the other calls).

i agree that calling userscript has no point to run before the actual call.

an other point to this area: should we handle error case of `close(i->backend_fd)`?

[the remark this answers is not in the document]
Comment (bAndie91) [reply execfuse.c]: should we handle error case of `close(i->backend_fd)`?

- maybe by pass `ret` to the script. i dunno. just wondering.

**Pull request merged**